api/pkg/apis/v1alpha1/utils/apiclient.go

```diff
@@ -205,7 +215,19 @@ func (a *apiClient) GetInstance(ctx context.Context, instance string, namespace
 		return ret, err
 	}
 
-	response, err := a.callRestAPI(ctx, "instances/"+url.QueryEscape(instance)+"?namespace="+url.QueryEscape(namespace), "GET", nil, token)
+	var name string
+	var version string
+	log.Infof("Symphony API GetInstance, instance: %s namespace: %s", instance, namespace)
+
+	parts := strings.Split(instance, ":")
+	if len(parts) == 2 {
+		name = parts[0]
+		version = parts[1]
+	} else {
+		return ret, errors.New("invalid target name")
+	}
```

Comment:
I would prefer having name and version in function parameters.

Reply:
use name and version explicitly can avoid caller passing wrong arguments and you can find a few places that you didn't change